app/components/app_vaccination_record_api_sync_status_component.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class AppVaccinationRecordAPISyncStatusComponent < ViewComponent::Base
+  SYNC_STATUS_COLOURS = {
+    synced: "green",
+    pending: "blue",
+    failed: "red",
+    cannot_sync: "orange",
+    not_synced: "grey"
+  }.freeze
+
+  def initialize(vaccination_record)
+    super
+    @vaccination_record = vaccination_record
+  end
+
+  def call
```

Comment:
I would suggest this method can be broken up in to various private methods on the class to improve readability. 

Just to give an example of the kind of thing I mean:

```rb
attr_reader :vaccination_record

delegate :nhs_immunisations_api_synced_at, :sync_status, to: :vaccination_record

def secondary_text(text) = tag.span(text, class: "nhsuk-u-secondary-text-color")

def last_synced_at
  return unless %i[not_synced cannot_sync].include?(status)

  secondary_text(
    "Last synced: #{
      if nhs_immunisations_api_synced_at?
        nhs_immunisations_api_synced_at.to_fs(:long)
      else
        "Never"
      end
    }"
  )
end

def additional_information_text
  ...
end

def call
  safe_join([sync_status_tag, additional_information_text, last_sync_at].compact, tag.br)
end
```